storage.py

```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
-    
+    def set(self, key, value):
+        if key in self.data:
```

Comment:
To access the array of keys you should write `self.data.keys`. Does the current version actually work?

Reply:
Yes, moreover, such a construction is used in the get() implementation.

Reply:
Ok, I haven't noticed it)